cosmos/carbon.json

```diff
@@ -99,9 +99,9 @@
       "coinDecimals": 8,
       "coinGeckoId": "switcheo",
       "gasPriceStep": {
-        "low": 769.2307692307693,
-        "average": 961.5384615384615,
-        "high": 1153.8461538461538
+        "low": 1,
+        "average": 1,
+        "high": 1
```

Comment:
Not sure if this is correct cos the gas fee calculation is as follows
`Gas Fee = gasPriceStep * gasCost`

In the case of low price step,
Gas Fee = 769.2307692307693 * 300000 = 2 SWTH (more than enough fees for standard transaction)

By changing `769.2307692307693` to `1`, sending tokens directly via Keplr form might not work anymore.
<img width="364" alt="Screenshot 2024-01-19 at 5 00 29 PM" src="https://github.com/chainapsis/keplr-chain-registry/assets/65583522/fe3fc6f2-3947-4550-9018-56edcfbf49bd">

i.e. if gas price step = 1,
Gas Fee = 1 * 300000 = 0.003 SWTH (not enough fees paid to send tokens => tx failed).

Think what would be better to recalculate the `low` value (such that tx fee = 1 SWTH), then apply the same value to `average` and `high`.

Calculation for `low` is as follows:
```typescript
gasPriceStep = (1 * 10^8) / 300000 (13 d.p. rounded up)
```
where SWTH decimals = 8 and Gas Cost on Keplr = 300000